dashboard/tailwind.config.cjs

```diff
@@ -1,72 +1,83 @@
-      roboto: ["Roboto", "serif"],
-      "roboto-slab": ["Roboto Slab", "sans-serif"],
-      inter: ["Inter", "serif"],
-      "sf-mono": [
-        "SFMono-Regular",
-        "Menlo",
-        "Monaco",
-        "Consolas",
-        "Liberation Mono",
-        "Courier New",
-        "monospace",
-      ],
-      monospace: ["monospace"],
-      poppins: ["Poppins", "sans-serif"],
-    },
-    fontWeight: {
-      thin: "100",
-      hairline: "100",
-      extralight: "200",
-      light: "300",
-      normal: "400",
-      semibold: "500",
-      extrabold: "600",
-      bold: "700",
-    },
-    extend: {
-      colors: {
-        grey: "#3d4048",
-        "light-grey": "#DCDDDF",
-        primary: "#1347FF",
-        secondary: "#ECEFF2",
-        muted: "#707583",
-        error: {
-          "border-color": "#DC3545",
-          background: "#F9D7DA",
-          color: "#842029",
+    content: [
+        "./index.html",
+        "./src/**/*.{js,ts,jsx,tsx}",
+        "./node_modules/flowbite/**/*.js",
+        "node_modules/flowbite-react/**/*.{js,jsx,ts,tsx}",
+    ],
+    theme: {
+        fontFamily: {
+            roboto: ["Roboto", "serif"],
+            "roboto-slab": ["Roboto Slab", "sans-serif"],
+            inter: ["Inter", "serif"],
+            "sf-mono": [
+                "SFMono-Regular",
+                "Menlo",
+                "Monaco",
+                "Consolas",
+                "Liberation Mono",
+                "Courier New",
+                "monospace",
+            ],
+            monospace: ["monospace"],
+            poppins: ["Poppins", "sans-serif"],
+        },
+        fontWeight: {
+            thin: "100",
+            hairline: "100",
+            extralight: "200",
+            light: "300",
+            normal: "400",
+            semibold: "500",
+            extrabold: "600",
+            bold: "700",
+        },
+        extend: {
+            colors: {
+                grey: "#3d4048",
+                "dark-gray": "#3B3D45",
+                "light-grey": "#DCDDDF",
+                primary: "#1347FF",
+                secondary: "#ECEFF2",
+                muted: "#707583",
+                error: {
+                    "border-color": "#DC3545",
+                    background: "#F9D7DA",
+                    color: "#842029",
+                },
+                failed: "#FC1683",
+                deployed: "#1FA470",
+                superseded: "#9195A1",
+                pending: "#5AB0FF",
+                danger: "#ff0072",
+                "text-danger": "#FC1683",
+                "text-warning": "#ffc107",
+                "text-success": "#A4F8D7",
+                upgradable: "#0d6efd",
+                warning: "#ffa800",
+                success: "#00c2ab",
+                "border-green": "#1BE99A",
+                "border-blue": "#5AB0FF",
+                "border-dark-blue": "#007bff",
+                "border-gray": '#dad8ce',
+                "gray-hover": "#E9ECEF",
```

Comment:
1. in tailwind you don't need to defined this, the way it works is that you define "green" and then you can use border-green, or hover:text-green
2. please don't define color names because you override tailwind default colors, name it like you did above, with some meaning (e.g - error, success, primary, secondary), this also goes for gray actually